WooCommerce/WooCommerceTests/ViewRelated/MainTabBarControllerTests.swift

```diff
@@ -199,79 +194,8 @@ final class MainTabBarControllerTests: XCTestCase {
         let selectedTabIndexAfterSiteChange = tabBarController.selectedIndex
 
         // Assert
-        XCTAssertEqual(selectedTabIndexBeforeSiteChange, WooTab.products.visibleIndex(isHubMenuFeatureFlagOn))
-        XCTAssertEqual(selectedTabIndexAfterSiteChange, WooTab.myStore.visibleIndex(isHubMenuFeatureFlagOn))
-    }
-
-    func test_when_receiving_a_review_notification_from_a_different_site_navigates_to_reviews_tab_and_presents_review_details() throws {
```

Comment:
do we not have this behavior with the hub menu? I think receiving a review notification from a different site should at least navigate to the menu tab 🤔

Reply:
As far as I understand it would seem that we do not navigate to any tab in particular, as we push the `ReviewDetailsViewController` directly into the stack, however, this is already being tested in the `ReviewsCoordinatorTests` under `test_when_receiving_a_review_notification_from_a_different_site_then_it_will_switch_the_current_site`.

The case for this test made sense when `hubMenu` was disabled, as we had a Reviews Tab instead and login involving the `MainTabBar` was in play:

- Navigate to the reviews tab. This no longer exists.
- Confirm the reviews tab had the correct visibleIndex(). This is no longer the case.
- Confirm the `topViewController` is an instance of `ReviewDetailsViewController`, which still happens.

Now we avoid any logic involving the `MainTabBar` and test the last item directly via the `ReviewsCoordinatorTests`, which is the same behaviour that when the hubMenu was enabled as a feature flag.

Reply:
I just thought it'd be nice to keep the original test with some updates for the menu tab, if possible 🙂 if it's not easy to wait for the review details, just verifying that it navigates to the menu tab is fine similar to the original test case.

Reply:
Gotcha! I played a bit with that but it seems I would need the `ReviewCoordinator`, which we plan to remove [here](https://github.com/woocommerce/woocommerce-ios/issues/7643) anyway, so for now I'll go ahead and wrap this one by just testing if navigates correctly to the menu tab, then when I work on the other PR I can check further if the test can be improved. Changed at https://github.com/woocommerce/woocommerce-ios/pull/7626/commits/8c782c98ee7d94b072accac1d48d76d9c230ca08